controller/node_controller.go

```diff
@@ -364,6 +364,12 @@ func (nc *NodeController) syncNode(key string) (err error) {
 	log := getLoggerForNode(nc.logger, node)
 
 	if node.DeletionTimestamp != nil {
+		// If V2 engine is enabled, we must clean up SPDK-related drivers
+		// before deleting the node. This ensures that SPDK devices can be reused on future node joins.
+		if err := nc.cleanupDisksBeforeNodeDeletion(node); err != nil {
+			nc.logger.WithError(err).Errorf("Cleanup spdk disk driver")
```

Comment:
Suggest a warning log instead, since error logs generally mean a real failure. Besides, suggest message change to `Failed to clean up disks during node deletion`